env.test.local

```diff
@@ -0,0 +1 @@
+DATABASE_URL="mysql://harry:potter@localhost:3306/projet3?serverVersion=8.0.32&charset=utf8mb4"
```

Comment:
It is nice of you to show publicly your DB credentials ... but maybe you should'nt

Reply:
ooops !